manifests/dht.toml

```diff
@@ -39,21 +39,25 @@ instances = { min = 16, max = 10000, default = 16 }
   n_bootstrap   = { type = "int", desc = "number of bootstrap nodes", unit = "int", default = "1" }
   bucket_size   = { type = "int", desc = "routing table bucket size", unit = "peers", default = "2" }
   n_find_peers  = { type = "int", desc = "number of peers to find", unit = "peers", default = "1" }
-  timeout_secs  = { type = "int", desc = "test timeout", unit = "seconds", default = "30" }
+  timeout_secs  = { type = "int", desc = "test timeout", unit = "seconds", default = "60" }
 
 # seq 1
 [[testcases]]
 name = "find-providers"
 instances = { min = 16, max = 250, default = 16 }
 
   [testcases.params]
```

Comment:
Looks like params in `dht.toml` `find-providers` diverged from the params in the code. Not sure how correct they are now, but I tried to align them.